libs/pages/environments/src/lib/feature/page-edit-deployment-rule-feature/page-edit-deployment-rule-feature.tsx

```diff
@@ -10,8 +9,9 @@ import { useEffect } from 'react'
 import { BaseLink } from '@console/shared/ui'
 import { ENVIRONMENTS_DEPLOYMENT_RULES_URL, ENVIRONMENTS_URL } from '@console/shared/router'
 import { dateToHours } from '@console/shared/utils'
+import PageEditDeploymentRule from '../../ui/page-edit-deployment-rule/page-edit-deployment-rule'
 
-export function EditDeploymentRule() {
+export function PageEditDeploymentRuleFeature() {
```

Comment:
Are these really pages or just component? I don' remember a dedicated route for this. Tell me if I'm wrong

Reply:
Yes they are pages, we just removed the routes for now